drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -312,13 +317,42 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
-    var mainTemplate = function(ctx) {
-        var problemTitle = ctx.show_title ? h('h2.problem-title', {innerHTML: ctx.title_html}) : null;
-        var problemHeader = ctx.show_problem_header ? h('h3.title1', gettext('Problem')) : null;
+    var itemFeedbackPopupTemplate = function(ctx) {
         var popupSelector = 'div.popup';
-        if (ctx.popup_html && !ctx.last_action_correct) {
+        var msgs = (ctx.feedback_messages || []).filter(function(msg) { return (msg) ? true : false; });
```

Comment:
@e-kolpakov IIUC, empty messages are also getting filtered out on the server (via `_suppress_empty_messages`). If that's the case you can remove the `.filter` portion from this line I think.